### PR TITLE
feat(tha-805): index lbp swap event logs

### PR DESCRIPTION
**Pull request by wesgraham**
logs information on lbp swap events to be returned to frontend on `api/lbp-swap-history` calls

tested and verified in personal and thala-labs namespaces

### CODE REVIEW

**Review by 0xbe1 — APPROVED** [review]
lgtm

File: src/processors/lbp.ts [hunk 1]
```diff
@@ -16,21 +16,44 @@ lbp
...
           getCoinDecimals(coin0)
         ),
         {
           poolId: getPoolId(event),
           dateString,
         }
       );
+    const swapAttributes = {
+      pair:
+          coin0.localeCompare(coin1) < 0
```
Comment (wjw12) [inline src/processors/lbp.ts:41]: https://github.com/ThalaLabs/thala-dapp/pull/726/files#r1113377171

only `${coin0}-${coin1}` is ok

Comment (wjw12) [reply src/processors/lbp.ts:41]: actually, it's better to use `coin0 + coin1 + creatorAddress` to uniquely identify a LBP. Two creators may launch the same pair (although unlikely, but possible) for example, a project messed up with first launch and re-launch

Comment (wesgraham) [reply src/processors/lbp.ts:41]: when we query for this metric in the front end we use multiple filters, querying based on pair + the creator address - will change coin0-1 and leave creator address as another unique field here!

**Pull request merged**